api/v1/users/routes.js

```diff
@@ -30,7 +43,9 @@ module.exports = [
         payload: {
           username: Joi.string().alphanum().min(3).max(30).required(),
           password: Joi.string().required(),
-          email: Joi.string().email().required()
+          email: Joi.string().email().required(),
+          created_at: Joi.date().timestamp(),
```

Comment:
we shouldn't validate created_at and updated_at. These are internal to routes and not user given values.